treeherder/webapp/api/performance_serializers.py

```diff
@@ -183,6 +185,42 @@ def update(self, instance, validated_data):
 
         return super().update(instance, validated_data)
 
+    def get_taskcluster_metadata(self, alert):
+        datum = PerformanceDatum.objects.filter(
+            signature=alert.series_signature,
+            repository=alert.series_signature.repository,
+            push_id=alert.summary.push,
+        ).first()
```

Comment:
```suggestion
        datum = Job.objects.filter(
            signature=alert.series_signature,
            repository=alert.series_signature.repository,
            push=alert.summary.push,
        )
```
Have you tried this approach?

Reply:
![image](https://github.com/mozilla/treeherder/assets/47977085/d1cb73b2-449d-46e8-92d8-deb7fe7ffc75)